man/crontab.5

```diff
@@ -205,6 +205,15 @@ hyphen.  The specified range is inclusive.  For example, 8-11 for
 an 'hours' entry specifies execution at hours 8, 9, 10, and 11. The first
 number must be less than or equal to the second one.
 .PP
+Randomization of the execution time within a range can be used.
+A random number within a range specified as two numbers separated with
+a tilde is picked.  The specified range is inclusive.
+For example, 6~15 for a 'minutes' entry picks a random minute
+within 6 to 15 range.  The random number is picked when crontab file is parsed.
+The first number must be less than or equal to the second one. You might omit
+the numbers specifying the range.  For example, ~ for a 'minutes' entry picks
```

Comment:
Please make it more explicit: `You might omit one or both numbers specifying the range.`